<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBAdmin</class>
 <widget class="QMainWindow" name="DBAdmin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>817</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Администратор базы данных Учет рабочих мест</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/128x128/ico/Database_128x128.png</normaloff>:/128x128/ico/Database_128x128.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_3">
    <property name="leftMargin">
     <number>2</number>
    </property>
    <property name="topMargin">
     <number>2</number>
    </property>
    <property name="rightMargin">
     <number>2</number>
    </property>
    <property name="bottomMargin">
     <number>2</number>
    </property>
    <property name="spacing">
     <number>2</number>
    </property>
    <item row="0" column="0">
     <widget class="QSplitter" name="splitterCentral">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>3</number>
      </property>
      <widget class="QGroupBox" name="groupBoxDB">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Базы данных</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QListView" name="listViewDataBases">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBoxUsers">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="title">
        <string>Пользователи</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="0">
         <widget class="QSplitter" name="splitterUser">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <widget class="QListView" name="usersView">
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
          </widget>
          <widget class="QTabWidget" name="userDataTabWidget">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="tabGeneral">
            <attribute name="title">
             <string>Общие</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <layout class="QGridLayout" name="gridLayout">
               <item row="5" column="0" colspan="3">
                <widget class="QGroupBox" name="groupBox">
                 <property name="title">
                  <string>Текущие права</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <property name="spacing">
                   <number>2</number>
                  </property>
                  <property name="leftMargin">
                   <number>2</number>
                  </property>
                  <property name="topMargin">
                   <number>2</number>
                  </property>
                  <property name="rightMargin">
                   <number>2</number>
                  </property>
                  <property name="bottomMargin">
                   <number>2</number>
                  </property>
                  <item>
                   <widget class="QPlainTextEdit" name="showGrantsUser">
                    <property name="lineWrapMode">
                     <enum>QPlainTextEdit::NoWrap</enum>
                    </property>
                    <property name="readOnly">
                     <bool>true</bool>
                    </property>
                    <property name="centerOnScroll">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="3" column="2">
                <widget class="QLineEdit" name="lineEditConfirmPass">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
               <item row="1" column="0" colspan="2">
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Хост:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="1" column="2">
                <widget class="QLineEdit" name="lineEditHost"/>
               </item>
               <item row="2" column="2">
                <widget class="QLineEdit" name="lineEditNewPass">
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                </widget>
               </item>
               <item row="0" column="0" colspan="2">
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Логин:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="3" column="0" colspan="2">
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Подтвердить пароль:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="2" column="0" colspan="2">
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Новый пароль:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item row="0" column="2">
                <widget class="QLineEdit" name="lineEditLogin"/>
               </item>
               <item row="4" column="0" colspan="3">
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <property name="spacing">
                  <number>2</number>
                 </property>
                 <item>
                  <spacer name="horizontalSpacer">
                   <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                   </property>
                   <property name="sizeHint" stdset="0">
                    <size>
                     <width>40</width>
                     <height>20</height>
                    </size>
                   </property>
                  </spacer>
                 </item>
                 <item>
                  <widget class="QPushButton" name="saveEditButton">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="font">
                    <font>
                     <weight>50</weight>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Сохранить</string>
                   </property>
                   <property name="icon">
                    <iconset resource="icons.qrc">
                     <normaloff>:/32x32/ico/save_accept_32x32.png</normaloff>:/32x32/ico/save_accept_32x32.png</iconset>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="revertEditButton">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="font">
                    <font>
                     <weight>50</weight>
                     <bold>false</bold>
                    </font>
                   </property>
                   <property name="text">
                    <string>Отмена</string>
                   </property>
                   <property name="icon">
                    <iconset resource="icons.qrc">
                     <normaloff>:/32x32/ico/save_delete_32x32.png</normaloff>:/32x32/ico/save_delete_32x32.png</iconset>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="tabPermission">
            <attribute name="title">
             <string>Права</string>
            </attribute>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>2</number>
             </property>
             <property name="topMargin">
              <number>2</number>
             </property>
             <property name="rightMargin">
              <number>2</number>
             </property>
             <property name="bottomMargin">
              <number>2</number>
             </property>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>База данных:</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="comboBoxDataBases">
                 <property name="font">
                  <font>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="editable">
                  <bool>false</bool>
                 </property>
                 <property name="currentText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBoxAccess">
               <property name="enabled">
                <bool>false</bool>
               </property>
               <property name="title">
                <string/>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <item>
                 <widget class="QRadioButton" name="isFullAccess">
                  <property name="text">
                   <string>Полный доступ</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="isReadOnly">
                  <property name="text">
                   <string>Только чтение</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="isRemoveGrants">
                  <property name="text">
                   <string>Удалить права</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QPushButton" name="setAccessButton">
                 <property name="enabled">
                  <bool>false</bool>
                 </property>
                 <property name="font">
                  <font>
                   <weight>50</weight>
                   <bold>false</bold>
                  </font>
                 </property>
                 <property name="text">
                  <string>Применить</string>
                 </property>
                 <property name="icon">
                  <iconset resource="icons.qrc">
                   <normaloff>:/16x16/ico/apply_16x16.png</normaloff>:/16x16/ico/apply_16x16.png</iconset>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>366</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>817</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="actionConnectToServer"/>
    <addaction name="actionDisconnectFromServer"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="actionCreateDB"/>
    <addaction name="actionDeleteDB"/>
    <addaction name="actionOptimizekDB"/>
    <addaction name="actionExportDB"/>
    <addaction name="actionImportDB"/>
    <addaction name="separator"/>
    <addaction name="actionCreateUser"/>
    <addaction name="actionDeleteUser"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionConnectToServer"/>
   <addaction name="actionDisconnectFromServer"/>
   <addaction name="separator"/>
   <addaction name="actionCreateDB"/>
   <addaction name="actionDeleteDB"/>
   <addaction name="actionOptimizekDB"/>
   <addaction name="actionExportDB"/>
   <addaction name="actionImportDB"/>
   <addaction name="separator"/>
   <addaction name="actionCreateUser"/>
   <addaction name="actionDeleteUser"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionConnectToServer">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/32x32/ico/connect_creating_32x32.png</normaloff>:/32x32/ico/connect_creating_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Подключить</string>
   </property>
   <property name="toolTip">
    <string>Подключение к серверу</string>
   </property>
  </action>
  <action name="actionDisconnectFromServer">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/32x32/ico/connect_no_32x32.png</normaloff>:/32x32/ico/connect_no_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Отключить</string>
   </property>
   <property name="toolTip">
    <string>Отключиться от сервера</string>
   </property>
  </action>
  <action name="actionCreateDB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/32x32/ico/add-to-database_32x32.png</normaloff>:/32x32/ico/add-to-database_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Создать БД</string>
   </property>
   <property name="toolTip">
    <string>Создать новую БД</string>
   </property>
  </action>
  <action name="actionDeleteDB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/32x32/ico/remove-from-database_32x32.png</normaloff>:/32x32/ico/remove-from-database_32x32.png</iconset>
   </property>
   <property name="text">
    <string>Удалить БД</string>
   </property>
   <property name="toolTip">
    <string>Удаление выбранной базы данных</string>
   </property>
  </action>
  <action name="actionCreateUser">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16x16/ico/add_user_16x16.png</normaloff>:/16x16/ico/add_user_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Создать  пользователя</string>
   </property>
   <property name="toolTip">
    <string>Создание нового пользователя</string>
   </property>
  </action>
  <action name="actionDeleteUser">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16x16/ico/delete_user_16x16.png</normaloff>:/16x16/ico/delete_user_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Удалить пользователя</string>
   </property>
   <property name="toolTip">
    <string>Удаление выбранного пользователя</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/16x16/ico/exit_16x16.png</normaloff>:/16x16/ico/exit_16x16.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="toolTip">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/32x32/ico/info_32x32.png</normaloff>:/32x32/ico/info_32x32.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="toolTip">
    <string>Сведения о программе</string>
   </property>
  </action>
  <action name="actionExportDB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/32x32/ico/Database-Upload.png</normaloff>:/32x32/ico/Database-Upload.png</iconset>
   </property>
   <property name="text">
    <string>Экспортировать базу</string>
   </property>
   <property name="toolTip">
    <string>Экспортировать базу данных</string>
   </property>
  </action>
  <action name="actionImportDB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/32x32/ico/download-database.png</normaloff>:/32x32/ico/download-database.png</iconset>
   </property>
   <property name="text">
    <string>Импортировать базу</string>
   </property>
   <property name="toolTip">
    <string>Импортировать базу данных</string>
   </property>
  </action>
  <action name="actionOptimizekDB">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/32x32/ico/accept-database.png</normaloff>:/32x32/ico/accept-database.png</iconset>
   </property>
   <property name="text">
    <string>Оптимизировать базу</string>
   </property>
   <property name="toolTip">
    <string>Выполнить оптимизацию базы данных</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>listViewDataBases</tabstop>
  <tabstop>usersView</tabstop>
  <tabstop>userDataTabWidget</tabstop>
  <tabstop>lineEditLogin</tabstop>
  <tabstop>lineEditHost</tabstop>
  <tabstop>lineEditNewPass</tabstop>
  <tabstop>lineEditConfirmPass</tabstop>
  <tabstop>saveEditButton</tabstop>
  <tabstop>revertEditButton</tabstop>
  <tabstop>showGrantsUser</tabstop>
  <tabstop>comboBoxDataBases</tabstop>
  <tabstop>isFullAccess</tabstop>
  <tabstop>isReadOnly</tabstop>
  <tabstop>isRemoveGrants</tabstop>
  <tabstop>setAccessButton</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>DBAdmin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>408</x>
     <y>346</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
